boutdata/data.py

```diff
@@ -1034,8 +1131,60 @@ def __init__(
                 self._datacachesize = 0
                 self._datacachemaxsize = self._caching * 1.0e9
 
+        if self._parallel is not False:
```

Comment:
Would this be better?

```suggestion
        if self._parallel:
```

Reply:
This was deliberate - I wanted to allow `parallel = 0` to be passed as a command-line argument (assuming that would only accept `int`) to mean 'use all available processors'. Not sure that's it's a great interface though... My `bout-squashoutput` command-line executable interface was broken (and I don't have the time to spend debugging it) so I stopped thinking about the command-line arguments.